core/build.gradle

```diff
@@ -43,6 +43,7 @@ dependencies {
     implementation 'commons-validator:commons-validator:1.6'
     implementation 'com.googlecode.libphonenumber:libphonenumber:8.12.13'
     implementation 'com.google.flogger:flogger:0.5.1'
+    testImplementation 'com.google.flogger:flogger-system-backend:0.5.1'
```

Comment:
The latest version of this library is 0.6, I suggest we use it (https://mvnrepository.com/artifact/com.google.flogger/flogger-system-backend).

Reply:
Let's upgrade all versions in a separate PR. We use 0.5.1 everywhere.